src/main/scala/scalismo/color/RGBA.scala

```diff
@@ -205,7 +205,7 @@ object RGBA {
     }
   }
 
-  private def toInt8(value: Double): Int = (value * 255.0).toInt
+  //private def toInt8(value: Double): Int = (value * 255.0).toInt
```

Comment:
Same as above: "I think we should not comment out functions to get rid of a compiler warning. Either we want to have the function because we develop a library and not an application so we might provide more functions as we are currently using in the project. We argue that the library can change, or the user might want to code package injections and we have forward-/backward-mappings in. Or we say we want only things we currently need."